xbmc/cores/VideoPlayer/DVDSubtitles/DVDSubtitlesLibass.cpp

```diff
@@ -258,15 +271,7 @@ void CDVDSubtitlesLibass::ApplyStyle(style subStyle, renderOpts opts)
     }
     else
     {
-      if (m_currentStyleId != ASS_NO_ID)
-      {
-        // Delete current style
-        ass_free_style(m_track, m_currentStyleId);
-        m_track->n_styles -= 1;
-      }
-      m_currentStyleId = ass_alloc_style(m_track);
-      m_track->default_style = m_currentStyleId;
-      style = m_track->styles + m_currentStyleId;
+      style = &m_track->styles[m_currentDefaultStyleId];
```

Comment:
now that the override ASS settings can be changed only when not in playing,
reuse always same style, how did you, should be fine

Reply:
Can you rephrase please? I did not understand. You mean this is wrong or right? Is forcing ass styles supposed to work during runtime playback and this breaks the functionality?

Reply:
my fear was that ass overrides settings might cause inconsistencies with style properties,
in short now i agree with this change

Reply:
I see, runtime tested and worked fine. Found a "bug" though (which might be in the settings internal implementation though):
- start playing a file and go to the settings window
- check that the ass override setting is disabled as it should
- hit "x" to stop the video while in this window
- check that the ass override setting is disabled which is wrong as nothing is playing
![image](https://user-images.githubusercontent.com/7375276/135709856-d5317895-35c3-4e97-b659-d5a3deb6af70.png)

- go back to another window and back to the settings window
- check that the setting is now enabled (correctly because nothing is playing)

Not a regression for sure, just something that likely was not tested found before

Reply:
maybe somewhere is needed to implement a way to receive callbacks like player status to update the list of settings, if it is curently displayed
probably is not implemented because my addition to check playback status is new